packages/rps/src/redux/game-v2/__tests__/scenarios.ts

```diff
@@ -78,7 +80,7 @@ const propsA = {
   theirWeapon: playerBWeapon,
 };
 
-export const localStatesA: Record<string, LocalState> = {
```

Comment:
Why did you remove this type?

Reply:
Because I discovered it was actually removing type-safety: I could do `localStatesA.blah` and it wouldn't complain. Also, now I'm using constructors, typescript knows that all the objects are `LocalState`s anyway.